KBase migration - create service (#226)
timescale/docs: timescale-cloud/create-a-service.md

# About services

> An MST service is an instance that corresponds to a cloud service provider tier.
> You can access all your services from the `Services` tab within your projects.
> Before you begin, make sure you have [signed in to MST][sign-up] and created your account.

Reviewer: ```suggestion
A Timescale Cloud service is an instance that corresponds to a cloud service provider tier.
You can access all your services from the `Services` tab within your projects.
Before you begin, make sure you have [signed up to Timescale Cloud][sign-up] and created your account.
```

Not sure about whether it should be "signed in" or "signed up" above given context.

Author: No, I think you're right, it's "sign up".

## Create a service

> When you have your account set up, you can log in to the [MST portal][timescale-cloud-portal] and create your first service.
> 
> ### Procedure: Creating a service
> 1.  [Log in to your MST account][mst-login]. By default, you start in the

Reviewer: ```suggestion
When you have your account set up, you can log in to the [Timescale Cloud portal][timescale-cloud-portal] and create your first service.

### Procedure: Creating a service
1.  [Log in to your Timescale Cloud account][mst-login]. By default, you start in the
```

    to the `Services` view while your service is created. The status indicator
    says `Rebuilding` while the service is created. It is ready for you to use
    when the indicator is green and says `Running`. This usually takes a couple
    of minutes, but different clouds can vary. You can click the service name in
    the list to see more information and make changes.
    <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/mst-buildservice-timescaledb.png" alt="Building the service"/>

<highlight type="tip">
During your Timescale Cloud trial, you have up to US$300 in credits to use. This
is enough credit to complete all our tutorials and run a few test projects.
</highlight>

If you're interested in learning more about pricing, visit the
[Timescale Cloud pricing calculator][timescale-pricing]. Or,
[contact us][contact] and we can walk you through the best configuration for
your use case.

## Access your database
The best way to access your new database is using the `psql` command line
utility. If you haven't already installed `psql`, you can follow
[these instructions][install-psql].

### Procedure: Accessing your database

> 1.  In the [MST account][mst-login] `Services` view, click the name of your new

Reviewer: ```suggestion
1.  In the [Timescale Cloud account][mst-login] `Services` view, click the name of your new
```

    Grafana service.
1.  On the service details page, take a note of the service URI for your service.
1.  Open a terminal, and use `psql` to connect to your service using the URI:
    ```bash
    psql -x "postgres://tsdbadmin:[password]@[hostname]:[port]/defaultdb?sslmode=require"
    ```
    When your connection is successful, you see a message like this:
    ```bash
    psql (12.2, server 11.6)
    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
    Type "help" for help.
    tsdbadmin@defaultdb=>
    ```
1.  You can verify that TimescaleDB is installed by running the `\dx` command
    at the `psql` prompt to see a list of all your PostgreSQL extensions.

Now you have your service up and running, you can try your
[first tutorial][getting-started].

[sign-up]: https://www.timescale.com/cloud-signup
[mst-login]: https://portal.timescale.cloud/login
[timescale-cloud-portal]: http://portal.timescale.cloud
[timescale-pricing]: https://www.timescale.com/products#cloud-pricing
[contact]: https://www.timescale.com/contact

> [getting-started]: /timescaledb/latest/getting-started/
> [install-psql]: /timescaledb/latest/how-to-guides/connecting/psql

Reviewer: ```suggestion
[getting-started]: /timescaledb/:currentVersion:/getting-started/
[install-psql]: /timescaledb/:currentVersion:/how-to-guides/connecting/psql
```
